aiplatform/src/main/java/aiplatform/CreateFeatureOnlineStoreFixedNodesSample.java

```diff
@@ -0,0 +1,102 @@
+
+public class CreateFeatureOnlineStoreFixedNodesSample {
+
+  public static void main(String[] args)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "YOUR_PROJECT_ID";
+    String featureOnlineStoreId = "YOUR_FEATURESTORE_ID";
+    int minNodeCount = 1;
+    int maxNodeCount = 2;
+    String location = "us-central1";
+    String endpoint = "us-central1-aiplatform.googleapis.com:443";
+    int timeout = 900;
+    createFeatureOnlineStoreFixedNodesSample(
+        project, featureOnlineStoreId, minNodeCount, maxNodeCount, location, endpoint, timeout);
+  }
+
+  static void createFeatureOnlineStoreFixedNodesSample(
+      String project,
+      String featureOnlineStoreId,
+      int minNodeCount,
+      int maxNodeCount,
+      String location,
+      String endpoint,
+      int timeout)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+
+    FeatureOnlineStoreAdminServiceSettings featureOnlineStoreAdminServiceSettings =
+        FeatureOnlineStoreAdminServiceSettings.newBuilder().setEndpoint(endpoint).build();
+
+    try (FeatureOnlineStoreAdminServiceClient featureOnlineStoreAdminServiceClient =
+        FeatureOnlineStoreAdminServiceClient.create(featureOnlineStoreAdminServiceSettings)) {
+
+      FeatureOnlineStore.Bigtable.Builder builderValue =
+          FeatureOnlineStore.Bigtable.newBuilder()
+              .setAutoScaling(
+                  FeatureOnlineStore.Bigtable.AutoScaling.newBuilder()
+                      .setMinNodeCount(minNodeCount)
+                      .setMaxNodeCount(maxNodeCount)
+                      .setCpuUtilizationTarget(60));
+      FeatureOnlineStore featureOnlineStore =
+          FeatureOnlineStore.newBuilder().setBigtable(builderValue).build();
+
+      CreateFeatureOnlineStoreRequest createFeatureOnlineStoreRequest =
+          CreateFeatureOnlineStoreRequest.newBuilder()
+              .setParent(LocationName.of(project, location).toString())
+              .setFeatureOnlineStore(featureOnlineStore)
+              .setFeatureOnlineStoreId(featureOnlineStoreId)
+              .build();
+
+      OperationFuture<FeatureOnlineStore, CreateFeatureOnlineStoreOperationMetadata>
+          featureOnlineStoreFuture =
+              featureOnlineStoreAdminServiceClient.createFeatureOnlineStoreAsync(
+                  createFeatureOnlineStoreRequest);
+      System.out.format(
+          "Operation name: %s%n", featureOnlineStoreFuture.getInitialFuture().get().getName());
+      System.out.println("Waiting for operation to finish...");
+      FeatureOnlineStore featureOnlineStoreResponse =
+          featureOnlineStoreFuture.get(timeout, TimeUnit.SECONDS);
+      System.out.println("Create FeatureOnlineStore Response");
+      System.out.format("Name: %s%n", featureOnlineStoreResponse.getName());
```

Comment:
please refactor the code sample method to return result(s). You can print it in the `main` method.
Printing results to `stdout` results in creating flaky tests.